draft-ietf-tls-record-limit.md

```diff
@@ -56,6 +56,11 @@ The `max_fragment_length` extension {{?RFC6066}} was designed to enable
 constrained clients to negotiate a lower record size.  However,
 `max_fragment_length` suffers from several design problems (see {{mfl-bad}}).
 
+A smaller protected record size is just one of many problems that a constrained
+implementation might need to address.  The `max_fragment_length` extension only
+addresses the memory allocation problem, it does not address limits of code
+size, or processing and bandwidth capacity.
```

Comment:
Well. It claims that it also addresses the bandwidth issue. However, it completely lacks any semantics or discusses the relationship between the two. 

IMHO the relationship between the RAM size problem and the bandwidth limitation is that the MFL provides the lower bound. However, using the MFL as a substitute using the (Path)MTU leads to problems since that information may change dynamically.

Reply:
I have expanded this a little, see what you think.